app/views/courses/_course_overview.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="objectives">
+  <h4>Learn</h4>
+
+  <ul class="objectives-list">
+    <% course.you_learn.each do |objective| %>
+      <li><%= objective %></li>
+    <% end %>
+  </ul>
+</div> <!-- /.objectives -->
+
+<div class="projects">
+  <h4>Projects</h4>
+
+  <ul class="projects-list">
+    <% course.you_build.each do |project| %>
+      <li><%= project %></li>
+    <% end %>
+  </ul>
+
+  <%= render 'see_more_link', course: course %>
```

Comment:
If its only displayed once in the new version, theres no need to display it twice.